### PR TITLE
922840 - Need to included the trademark symbol ®

### PR DESCRIPTION
**Pull request by Hemalatha-SF4675**
Hi team,

I have included the trademark symbol ® with the term Essential Studio, as shown below:
Essential Studio®

Regards,
Hemalatha

### CODE REVIEW

File: MAUI/TextInputLayout/Events.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
```
Comment (PrithisIyyappan) [inline MAUI/TextInputLayout/Events.md:2]: Why tags are modified? The trademark symbol is improper.

Comment (Hemalatha-SF4675) [reply MAUI/TextInputLayout/Events.md:2]: I am already facing MD tag validation errors. There seems to be a mistake in the highlight tag on line number 28.
So I fix it by changing the tags

File: MAUI/Backdrop/overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
```
[the remark this answers is not in the document]
Comment (Hemalatha-SF4675) [reply MAUI/Backdrop/overview.md:1]: Updated

**Comment by kalirajanganeshan** [comment]
As per @PaulAndersonS  instruction this got force merged

**Pull request merged**